pallets/runtime/src/dividend.rs

```diff
@@ -488,6 +488,7 @@ mod tests {
 
     use crate::{
         asset::{AssetType, SecurityToken, TickerRegistrationConfig},
+        cdd_check::CddChecker,
```

Comment:
Is it possible to make a new crate in `pallets`? Otherwise we are overloading the runtime crate with modules that don't depend on anything within it. Alternatively, this module could be added to the identity crate.

Reply:
Since this module depends on the multisig, if we were to move this to the identity module then we'll have to create another sub trait in the multisig module that has the function we need because we can't make the identity module depend on the multisig module due to circular dependency.

I think having it within the runtime crate is ok. There's like only 30 lines of significant code in there and creating a separate crate for just those 30 lines doesn't make much sense to me due to the increased maintenance cost of crates.

Reply:
Agreed, it's not worth writing extra lines.